bin/phoenix_queryserver_utils.py

```diff
@@ -131,11 +132,16 @@ def setPath():
     phoenix_queryserver_classpath = os.path.join(current_dir, "../lib/*")
 
     global phoenix_client_jar
-    phoenix_client_jar = find(PHOENIX_CLIENT_JAR_PATTERN, phoenix_class_path)
+    phoenix_client_jar = find(PHOENIX_CLIENT_EMBEDDED_JAR_PATTERN, phoenix_class_path)
     if phoenix_client_jar == "":
-        phoenix_client_jar = find(OLD_PHOENIX_CLIENT_JAR_PATTERN, phoenix_class_path)
+        phoenix_client_jar = findFileInPathWithoutRecursion(PHOENIX_CLIENT_EMBEDDED_JAR_PATTERN, os.path.join(current_dir, ".."))
```

Comment:
I switched the ordering around so that we search for the each pattern in both locations, instead of searching one location for each pattern, then searching the other one for each pattern.